tests/tailor.js

```diff
@@ -56,6 +57,24 @@ describe('Tailor', () => {
                     return Promise.reject(error);
                 }
             },
+            handledTags: ['delayed-fragment'],
+            handleTag: (request, tag, options, context) => {
+                if (tag.name === 'delayed-fragment') {
+                    const st = processTemplate(request, options, context);
+                    setTimeout(() => {
+                        st.end({
+                            name: 'fragment',
+                            attributes: {
+                                async: true,
```

Comment:
Before, since the fragment's async + also located in handleTag, there would be an error, cause one can't write to async channel after it's ended (which happens on 'finish' events which is all fine, cause we stop writing to the original resultStream right after pushing all parsed chunks).